tests/integration/test_force_trigger.py

```diff
@@ -129,23 +173,313 @@ async def test_trigger_on_resume(
         assert len(submitted_tasks) == 0
 
         # manually trigger 1/x - it not should be submitted
-        schd.pool.force_trigger_tasks(['1/x'], [1], on_resume=True)
+        await run_cmd(
+            force_trigger_tasks(schd, ['1/x'], ["1"], on_resume=True))
         schd.release_tasks_to_run()
         assert len(submitted_tasks) == 0
 
         # manually trigger 1/y - it should not be submitted
         # (queue limit reached)
-        schd.pool.force_trigger_tasks(['1/y'], [1], on_resume=True)
+        await run_cmd(
+            force_trigger_tasks(schd, ['1/y'], ["1"], on_resume=True))
         schd.release_tasks_to_run()
         assert len(submitted_tasks) == 0
 
         # manually trigger 1/y again - it should not be submitted
         # (triggering a queued task runs it)
-        schd.pool.force_trigger_tasks(['1/y'], [1], on_resume=True)
+        await run_cmd(
+            force_trigger_tasks(schd, ['1/y'], ["1"], on_resume=True))
         schd.release_tasks_to_run()
         assert len(submitted_tasks) == 0
 
         # resume the workflow, both tasks should trigger now.
         schd.resume_workflow()
         schd.release_tasks_to_run()
         assert len(submitted_tasks) == 2
+
+
+async def test_trigger_group(
+    flow, scheduler, run, complete, log_filter
+):
+    """Test trigger of a sub-graph of future, then past, tasks.
+
+    It should satisfy off-group task and xtrigger prerequisites automatically.
+
+    """
+    cfg = {
+        'scheduling': {
+            'xtriggers': {
+                'xr': 'xrandom(0)'  # never satisfied
+            },
+            'graph': {
+                'R1': """
+                    # upstream:
+                    x => a
+
+                    # sub-graph for group trigger:
+                    a => b & c => d
+
+                    # downstream:
+                    d => e => y
+
+                    # off-group prerequisites:
+                    @xr => x  # stop the flow starting
```

Comment:
See https://github.com/cylc/cylc-flow/pull/6395/files#r2210436921
```suggestion
                    ext_trigger => x
                    @xr => x  # stop the flow starting
```

Reply:
done